pkg/threescale/builder_test.go

```diff
@@ -0,0 +1,44 @@
+package threescale
+
+import (
+	"net/http"
+	"testing"
+)
+
+func TestClientBuilder_BuildSystemClient(t *testing.T) {
+	builder := NewClientBuilder(http.DefaultClient)
+
+	_, err := builder.BuildSystemClient("invalid.due.to.no.scheme")
+	if err == nil {
+		t.Errorf("expected failure due to no scheme provided")
+	}
+
+	// todo : this test is commented while https://github.com/3scale/3scale-porta-go-client/pull/19 remains
+	// open and the dependency not updated
```

Comment:
it is now closed and the dependency can be updated, no? :)

Reply:
it can but lets wait and see if anything else raises up from the dead :)

If nothing is obvious when we merge the rest of these changes I'll create a release and we can use that